web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -197,20 +200,20 @@
   {/if}
 
   <section class="mx-4 mt-10" style:display={!isOwner && description === '' ? 'none' : 'block'}>
-    {#if !isOwner || api.isSharedLink}
-      <span class="break-words">{description}</span>
-    {:else}
```

Comment:
Why did you get rid of this span and convert it back to a placeholder? Did you test a multi-line description in a shared link, specifically going from short to long to see if it auto grows?

Reply:
Yes it works fine.

Original asset:

![Screenshot from 2024-01-20 02-44-51](https://github.com/immich-app/immich/assets/74269598/6346cc7b-467f-4688-a8d1-5cd86ee15cd6)

| Before | After |
| :---: | :---: |
| ![Screenshot from 2024-01-20 02-48-16](https://github.com/immich-app/immich/assets/74269598/9d723fd1-940e-43e6-bd9d-6e9bd0192c9f) | ![Screenshot from 2024-01-20 02-48-29](https://github.com/immich-app/immich/assets/74269598/335cf839-cb44-4716-b405-b8301d99a3d3) |